### ISSUE TITLE
[Request] Update to 1.10

### ISSUE THREAD

**Issue by Nuinethir** [Nuchaz/BiblioCraft#368]
A number of modders are starting to make the move to 1.10 and I was curious (as many others probably are) whether BiblioCraft will also make the jump. I consider this plugin an essential, a fundamental, due to the variety of craftable furniture and other objects that still fit within the realm of being considered "vanilla". I miss it, yet after two years, I am making the migration away from 1.7.10.

Thank you for taking the time to read this and I look forward to using your mod again. :)

**Comment by Nuchaz** [comment]
I have a port in progress. I am getting close to beta testing my port to 1.8.9 (which was a rewrite of like 70-80% of entire mod) then I plan to update to 1.9.x and on to 1.10.x right after.

**Comment by Nuinethir** [comment]
Sweet! Thanks for your hard work! Looking forward to it.

**Comment by Beardedlizzy** [comment]
I am also looking forward to it.
---Random person who likes the mod

**Comment by ldittmar81** [comment]
Hello Nunchaz. What about a alpha version for 1.10.2 ... I know there is a problem with forge, but I think your 1.10.x version ist ready for this. What do you think about it?

**Comment by Iteries** [comment]
I am an v7.10 Biblocrafter now I am v10.2  looking forward to when Biblocraft updates.  Biblocraft is the decor I am in hope for and missing soooo much and the only reason I haven't used something else close to it.
Sad part is that MC moving ever forward quickly and v11.1 about/is here.  If can help out I would but the only help is this:
KEEP UP THE GREAT WORK!  Many are looking forward to your Crafting!

**Comment by hron84** [comment]
I really miss your bookcases in my 1.10 worlds. Enchantment books took over my loot chests :disappointed: I hope we can see your awesome bookcases soon! :smile:

**Comment by FormalHellhound** [comment]
There hasn't been any news on progress since July. This mod should be open sourced so that someone can continue developement, or someone should be brought in. This mod is a great mod and I would hate to see it die, but it is now almost 4 versions behind, and is beginning to feel like it has been abandoned.

**Comment by Nuchaz** [comment]
I promise it isn't abandoned. I just haven't had much time due to college and other responsibilities. But I do work on it when I can. The problem is I am stuck on this issue: https://github.com/MinecraftForge/MinecraftForge/issues/3173

I have everything working except item rendering, which makes it super frustrating to use. I have a working prototype in 1.9.4 and 1.10.x, but yeah, that bug is frustrating. I can't figure out what is going on. I forked forge and have been trying to figure out whats up but nothing I've tried has worked yet. 

I started porting to 1.11 as well but I still have a number of errors to fix. I expect the same issue to be present, but will have to finish updating to be sure.
